lighthouse-core/report/v2/templates.html

```diff
@@ -18,9 +18,231 @@
     <details class="lighthouse-score__header">
       <summary class="lighthouse-score__snippet">
         <span class="lighthouse-score__title"><!-- fill me --></span>
-        <div class="lighthouse-score__arrow" title="See audits"></div>
+        <div class="lighthouse-toggle-arrow" title="See audits"></div>
       </summary>
       <div class="lighthouse-score__description"><!-- fill me --></div>
     </details>
   </div>
 </template>
+
+<!-- Lighthouse header -->
+<template id="tmpl-lighthouse-heading">
+  <style>
```

Comment:
For these  one-time-use partials....instead of creating separate .css file for each (or adding to the main stylesheet), I chose to keep related styles together with the markup that requires them. One nice thing about this is that these styles don't get added to the page unless a renderer uses the template. So if devtools renders its own header, we're not penalized.